lua/neotest-vitest/init.lua

```diff
@@ -345,7 +304,7 @@ function adapter.build_spec(args)
     "--reporter=verbose",
     "--reporter=json",
     "--outputFile=" .. results_path,
-    "--testNamePattern=" .. testNamePattern,
+    "'--testNamePattern=" .. testNamePattern .. "'",
```

Comment:
The single quotes break the test execution in my environment. The whole file tests are executed ignoring the testNamePattern. Is there a special reason you added them?

Reply:
hey @AbaoFromCUG did you take a look at this comment?